tests/ca_setup_test.go

```diff
@@ -85,7 +85,8 @@ func TestCASetup(t *testing.T) {
 			defer cancel()
 			resp, err := service.URLHasValidCert(ctx, &central.URLHasValidCertRequest{Url: c.url})
 			require.NoError(t, err)
-			assert.Equal(t, c.expectedResp, resp.GetResult(), "received resp: %+v. %s", resp, c.additionalMessage)
+			t.Logf("received resp: %+v. %s", resp, c.additionalMessage)
+			assert.Equal(t, c.expectedResp, resp.GetResult())
```

Comment:
Is there a reason for logging the message now instead of keeping the previous logic?

Reply:
The comparison is for an enum, so no need for protocompat here.
I also don't see a need to change the logic.

Reply:
This was a false positive as some args were from proto. Fixed that!